### ISSUE TITLE
the parameter file

### ISSUE THREAD

**Issue by Jacksonpe** [SCAU-AIUS/SLAM-for-agricultural-equipment#1]
Hello, may I ask, the parameter file (.yaml) of the camera, can you put it?

**Comment by SCAU-AIUS** [comment]
> Hello, may I ask, the parameter file (.yaml) of the camera, can you put it?

Of course. We found that the camera parameters file was missing and has now been updated in /dataset/D435i.yaml
